### PR TITLE
Redo the project as a docker-maven-plugin project.

### PR DESCRIPTION
**Pull request by cunningt**
Provides property substitution and a pom.xml so we can align the version coming from keycloak-customizations within the pipeline.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+      <distribution>repo</distribution>
+    </license>
+  </licenses>
+
+  <!-- including A developer as it's required by the maven poms going into
+    central -->
+  <developers>
+    <developer>
+      <id>geeks</id>
```
Comment (zregvart) [inline pom.xml:51]: 💯

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+      <name>Red Hat iPaaS Development Team</name>
+      <organization>Red Hat</organization>
+      <organizationUrl>https://redhat.com/</organizationUrl>
+    </developer>
+  </developers>
+
+  <scm>
+    <connection>scm:git:git@github.com:redhat-ipaas/keycloak-customizations.git</connection>
+    <developerConnection>scm:git:git@github.com:redhat-ipaas/keycloak-customizations.git</developerConnection>
+    <url>http://github.com/redhat-ipaas/keycloak-customizations/</url>
```
Comment (zregvart) [inline pom.xml]: should this point to `syndesisio/keycloak-openshift` ?

**Review by zregvart — CHANGES_REQUESTED** [review]
perhaps the files root that are copied over to `src/main/docker` can be now removed from the root?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.redhat.ipaas.keycloak</groupId>
```
Comment (jimmidyson) [inline pom.xml]: `io.syndesis` please. This should be the same top level group ID for all projects in the syndesisio org.

**Review by zregvart — APPROVED** [review]
👍 thanks!

**Pull request merged**